### DISCUSSION TITLE
Uncheck status field of a model in Fields Permissions of a role result in forbidden error

### DISCUSSION THREAD

**Discussion by BaptisteHouques** [directus/directus#13109] [Feature Requests]
### Describe the Bug

In Roles & Permissions, when creating a role having READ **custom** permission and unchecked status default field of the model and create a user with this role result in showing a forbidden error when this user is in the model interface.

### To Reproduce

1. Create a model/collection with the status field and an other (like input).
2. Create a role with custom READ access for this model/collection.
3. In Field Permissions check ID & the other field but uncheck the status field.
4. Create a user and assign the previously created role.
5. Log in with this new user and go to the model/collection created
6. This result (for me) in a Unexpected Error.

### Errors Shown

{
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}[
![forbidden](https://user-images.githubusercontent.com/65606239/165532097-69aa0b83-0579-426d-bc1b-52166134096b.PNG)
![log](https://user-images.githubusercontent.com/65606239/165532587-f6330cb9-b3ed-480f-bbf8-c15cbf04fd60.PNG)
](url)

### What version of Directus are you using?

9.9.1

### What version of Node.js are you using?

16.14.2

### What database are you using?

MySQL 5.7.36

### What browser are you using?

Chrome

### How are you deploying Directus?

Locally

**Comment by br41nslug** [comment]
This results from the "App Archive Filter" which is enabled by default when adding a status field. While this might not be intuitive I don't think this is a bug.
For the archive filter to work a status is required to filter but perhaps a warning when creating permissions that won't work with this filter would be appropriate.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
